spec/ffi/function_spec.rb

```diff
@@ -64,14 +64,14 @@ class << self; self; end
 
   it 'can wrap a blocking function' do
```

Comment:
This test is broken; timing for a blocking call cannot be relied upon within any sane window. see #397